Cargo.lock

```diff
@@ -3108,7 +3108,7 @@ dependencies = [
 [[package]]
 name = "librocksdb_sys"
 version = "0.1.0"
-source = "git+https://github.com/SpadeA-Tang/rust-rocksdb.git?branch=fix-sealed-chaos#f5121f48a1543c5d576ad7964c617f30f79a3d66"
```

Comment:
How it's checked-in at first place?

Reply:
seems ci does not check the consistency between cargo.toml and cargo.lock